lib/protobuf/enum.rb

```diff
@@ -283,12 +283,13 @@ def to_int
       tag.to_int
     end
 
-    ##
     # This fixes a reflection bug in JrJackson RubyAnySerializer that does not
-    #     render Protobuf enums correctly because to_json is not defined.
+    # render Protobuf enums correctly because to_json is not defined. It takes
+    # any number of arguments to support the JSON gem trying to pass an argument.
+    # NB: This method is required to return a string and not an integer.
     #
-    def to_json
-      to_i
+    def to_json(*)
+      to_s
```

Comment:
We actually debated this and realized that it would make more sense for this to be an integer. Perhaps I should have released this as a minor version bump. Were you previously relying on a string value from this method?

Reply:
It’s the method signature required by the JSON gem - if you checkout the branch, switch it back to to_i, and run the specs you’ll get a TypeError.